src/video/domain.ts

```diff
@@ -79,6 +79,12 @@ export interface InputOverlaySettings {
   opacity?: string;
 }
 
+export interface GeneratedSubtitlesSettings {
+  name?: string;
+  passthrough?: string;
+  language_code?: string;
+}
```

Comment:
interestingly none of these are actually required. This request:

```json
{
	"playback_policy": [
		"public"
	],
	"input": [
		{
			"url": "https://muxed.s3.amazonaws.com/leds.mp4",
			"generated_subtitles": [
				{}
			]
		}
	]
}
```

Will result in a text track being created with default `language_code: "en"` and default `name: "English CC"`

```
			{
				"type": "text",
				"text_type": "subtitles",
				"text_source": "generated_vod",
				"status": "preparing",
				"name": "English CC",
				"language_code": "en",
				"id": "zHkS8uZkPyY5v45ixKB3pgF8DEZyJrj5P2Nd7bZg2GidHOAyQX6rFQ"
			}
```